packages/react-docs/RELEASE-NOTES.md

```diff
@@ -37,18 +37,21 @@ Packages released:
   - Allow count badge to be hidden in checkbox select ([#3976](https://github.com/patternfly/patternfly-react/pull/3976))
   - Added logic for disabled default options for typeahead ([#3895](https://github.com/patternfly/patternfly-react/pull/3895))
   - Fixed group select options in single variant ([#3838](https://github.com/patternfly/patternfly-react/pull/3838))
-- **Table:**
+### Table
   - Added column management demo ([#3942](https://github.com/patternfly/patternfly-react/pull/3942))
-  - Fix forward ref types ([#3919](https://github.com/patternfly/patternfly-react/pull/3919))
+  - Fixed forward ref types ([#3919](https://github.com/patternfly/patternfly-react/pull/3919))
 
 ### Other
-- Create unified react-core dist ([#3971](https://github.com/patternfly/patternfly-react/pull/3971))
-- In-house focusTrap ([#3967](https://github.com/patternfly/patternfly-react/pull/3967))
-- Remove a few usages of prop-types ([#3968](https://github.com/patternfly/patternfly-react/pull/3968))
-- Release docs to NPM for patternfly-org ([#3941](https://github.com/patternfly/patternfly-react/pull/3941))
-- Fix tippy types, remove copyTS ([#3940](https://github.com/patternfly/patternfly-react/pull/3940))
-- Create variables by file in react-tokens ([#3896](https://github.com/patternfly/patternfly-react/pull/3896))
-- Remove unused packages from repo ([#3916](https://github.com/patternfly/patternfly-react/pull/3916))
+- **Docs:** 
+  - Release docs to NPM for patternfly-org ([#3941](https://github.com/patternfly/patternfly-react/pull/3941))
+- **Build:**
+  - Created unified react-core dist ([#3971](https://github.com/patternfly/patternfly-react/pull/3971))
+  - In-house focusTrap ([#3967](https://github.com/patternfly/patternfly-react/pull/3967))
+  - Remove unused packages from repo ([#3916](https://github.com/patternfly/patternfly-react/pull/3916))
```

Comment:
Removed

Reply:
I accidentally committed this file. I'll have to update it in the next PR once I know the new version numbers that need to be released.

Reply:
For now I think the changes it makes are okay... can you approve? It's the thing I'm going to work on right after this gets merged and there's newly released versions. See my message in Slack for the release timeline.